packages/binding-coap/src/coap-server.ts

```diff
@@ -32,11 +32,15 @@ import { Server, createServer, registerFormat, IncomingMessage, OutgoingMessage
 import slugify from "slugify";
 import { Readable } from "stream";
 import { MdnsIntroducer } from "./mdns-introducer";
+import { PropertyElement, DataSchema } from "wot-thing-description-types";
 
 const { debug, warn, info, error } = createLoggers("binding-coap", "coap-server");
 
 type CoreLinkFormatParameters = Map<string, string[] | number[]>;
 
+// TODO: Move to core?
+type AugmentedInteractionOptions = WoT.InteractionOptions & { formIndex: number };
```

Comment:
Why doesn't `WoT.InteractionOptions` have `formIndex` already? Current scripting API already mention it: https://w3c.github.io/wot-scripting-api/#dom-interactionoptions-formindex also in TS file they are indeed present:https://github.com/w3c/wot-scripting-api/blob/06a31b1efb06d8b1ec996cd18dab25c71a98b2b1/typescript/scripting-api/index.d.ts#L175.

Reply:
Agree, let look into it in a follow-up. Created #1039

Reply:
Thank you daniel, we can resolve this issue.